README.md

```diff
@@ -5,18 +5,30 @@
 # OpenShift Ansible
 
 This repository contains [Ansible](https://www.ansible.com/) roles and
-playbooks to install, upgrade, and manage
-[OpenShift](https://www.openshift.com/) clusters.
-
-**Note**: the Ansible playbooks in this repository require an RPM
-package that provides `docker`. Currently, the RPMs from
-[dockerproject.org](https://dockerproject.org/) do not provide this
-requirement, though they may in the future. This limitation is being
-tracked by
-[#2720](https://github.com/openshift/openshift-ansible/issues/2720).
-
-## Getting the correct version
-When choosing an openshift release, ensure that the necessary origin packages
+playbooks to install, upgrade, and manage the Origin Kubernetes Distribution in a fully managed container orchestration platform, on an existing computing infrastructure.  For an introduction to openshift [OpenShift](https://www.openshift.com/).
+
+## For ansible or openshift newcomers
+
+If you've used kubernetes before, then openshift-ansible is the OKD equivalent to Kubeadm (rather then, say, KOPS): It configures the entire cluster on top of existing infrastructure, rather managing the spinning up of infrastructure *for you*.
+
+### Bring your own VMs or Hardware.
+
+OpenShift ansible can easily 'just work' on a single node, so a "real cluster" isn't necessary for your first installation.  Nevertheless: You need to provide the compute
+instance for ansible to run against.
+
+We clarify 'computing infrastructure' above because openshift ansible does not provision
```

Comment:
>openshift ansible does not provision infrastructure for you in any manner

It can though, dynamic playbooks for GCP and AWS do that

Reply:
Where do those playbooks live ? Are they using terraform or vagrannt ?  I didn't see any cloud APIs for creating VM infrastructure in this Repo.   Can you point me at it?  Actually it would save me alot of time, as we maintain our own terraform based wrappers to openshift ansible on origin at my current company ...